lametro/utils.py

```diff
@@ -145,3 +146,21 @@ def trace_function(frame, event, arg):
         sys.settrace(None)
 
     return resp
+
+
+def get_wagtail_document(queryset, static_path, static_title):
+    """
+    Get the most relevant wagtail document. If none found,
+    return a dict with the url and title for the corresponding static file.
```

Comment:
A modular helper function that we can use to process any other files we'd like to have in templates